.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - 2.7
+  - 3.4
```

Comment:
I don't actually know, 3.4 was just what ships in Jessie at the moment and it seemed like a reasonable place to start. It might as well track latest stable Python, that just isn't my dev and functional testing environment.